shared/macros/fixtext.jinja

```diff
@@ -18,6 +18,22 @@ Add or update the following file system rule to <tt>{{{ rule_path }}}</tt>:
 The audit daemon must be restarted for the changes to take effect.
 {{% endmacro %}}
 
+{{#
+    Generate a fixtext for audit rules for file deletion events
+
+:param syscall: Sycall name
+:type path: str
+
+#}}
+{{% macro fixtext_audit_rules_file_deletion_events(syscall) -%}}
+Configure the audit system to generate an audit event for any successful/unsuccessful use of the "rename" command by adding or updating the following rules in the "/etc/audit/rules.d/audit.rules" file:
+
+-a always,exit -F arch=b32 -S {{{ syscall }}} -F auid>=1000 -F auid!=unset -k delete
+-a always,exit -F arch=b64 -S {{{ syscall }}} -F auid>=1000 -F auid!=unset -k delete
+
+The audit daemon must be restarted for the changes to take effect.
+{{% endmacro %}}
+
```

Comment:
This is a question: I have taken this from the RHEL 8 spreadsheet. But, the actual bash remediation has about 329 lines of code. How to transform these complex bash remediations to a fixtext?

Reply:
In my opinion the bash remediation should be used to support us when writing the fix text. It doesn't necessarily mean we need to bring every action found there. Usually reusing the text from the RHEL8 STIG should be enough with minor adjustments. Always try to give preference to the latest RHEL8 STIG version as it may contain bugfixes and new content: https://complianceascode.github.io/content/tables/table-rhel8-stig-manual.html

Reply:
Great, makes sense, thanks for the table I didn't know about it.

In this specific example the table has all audit deletion events in a single rule. There is:

> Configure the audit system to generate an audit event for any successful/unsuccessful use of the "rename", "unlink", "rmdir", "renameat", and "unlinkat" system calls by adding or updating the following rules in the "/etc/audit/rules.d/audit.rules" file:

Is it expected that we will have it split into 5 separate rules 1 rule for 1 syscall?

Reply:
Maybe we can add a text to the fixtext/ocil of these rules like:

`It's allowed to group this system call within the same line as "rename", "unlink", "rmdir", "renameat", and "unlinkat"`

Or some different text to achieve this interconnection goal

Reply:
Thanks for explanation. I will add your proposal to the fixtext.

Reply:
We will probably reuse that macro for other audit rules as well, but improvements can be made in further pull requests.